stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/pageTitle"
+        style="@style/TextAppearance.MaterialComponents.Headline4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="24dp"
```

Comment:
Our `dimens.xml` contains some commonly used spacings (e.g. 8, 16, 24, etc.). Maybe let's reuse them?

Reply:
Do we expect to change those spacings at some point, and when that happens, for all of their usages to change together?

I'm always sceptical about dimension constants, because they often just mean "24dp" and not "this same medium padding everywhere it's used". They're rarely a good abstraction.

Reply:
👍 good point, I don't think we can expect such changes